Formula/perl.rb

```diff
@@ -4,6 +4,7 @@ class Perl < Formula
   url "https://www.cpan.org/src/5.0/perl-5.34.0.tar.xz"
   sha256 "82c2e5e5c71b0e10487a80d79140469ab1f8056349ca8545140a224dbbed7ded"
   license any_of: ["Artistic-1.0-Perl", "GPL-1.0-or-later"]
+  revision 1
```

Comment:
This is dangerous in combination with skipping dependents, since I think we have dependents that link to perl via a `Cellar` path for some reason.

Reply:
We should do dependent testing but skip fixing things that are not direct consequences of this change.  Also, we should make an issue to fix whatever is using the `Cellar` because that is very problematic for upgrades!